web/src/lib/components/shared-components/side-bar/side-bar.svelte

```diff
@@ -69,9 +73,30 @@
 			};
 		}
 	};
+
+	const handleResize = () => {
+		if (innerWidth > 768) {
+			isCollapsed = false;
+		} else {
+			isCollapsed = true;
+		}
+	};
+
+	onMount(() => {
+		handleResize();
+		window.addEventListener('resize', handleResize);
+	});
```

Comment:
The event listener is added each time the component mounts, but it is never removed. Svelte provides a special element that can manage window event listeners for you. An example that also prevents using the global `innerWidth` variable:

```svelte
<script lang="ts">
	let innerWidth: number;

	onMount(() => {
		handleResize();
	});
</script>

<svelte:window on:resize={handleResize} bind:innerWidth />
```

Hardcoding the `md` breakpoint of 768px isn't that nice and the value might not make sense for some people without additional context. Also `@media (min-width: 768px)` means more or equal to `768` so you have an off-by-one error.

Alternatively this can probably be implemented with CSS only, using the Tailwind's `group` class. That will also prevent the layout shifting on desktop, because JS is executed later than CSS.

Reply:
1) Done in af168cc99f47665b893538876e0498d807a71b7b
2) I think for now hardcoding the breakpoint is the easiest way. I just oriented my self on how google has done it (they used max-width: 1007px).
3) I don't have used the Tailwind's `group` class up to now and don't know how to use it. I will try to read me in, mabye i can implement it that way, if not feel free to give it a go yourself.

P.S. Thanks for your feedback. As that is my first pull request, I am very interested and motivated in fixing the issues you mentioned.